lib/avrora/storage/registry.ex

```diff
@@ -33,7 +33,8 @@ defmodule Avrora.Storage.Registry do
          {:ok, id} <- Map.fetch(response, "id"),
          {:ok, version} <- Map.fetch(response, "version"),
          {:ok, schema} <- Map.fetch(response, "schema"),
-         {:ok, schema} <- Schema.parse(schema) do
+         {:ok, references} <- extract_references(response),
+         {:ok, schema} <- Schema.parse(schema, make_references_lookup_fun(references)) do
```

Comment:
Please rename `make_references_lookup_fun` to `make_reference_lookup_fun` (see below)